src/parser.js

```diff
@@ -638,6 +650,38 @@ export default class Parser {
     return schemes;
   }
 
+  handleSwaggerDefinitions(definitions) {
+    const { Category } = this.minim.elements;
+    const generator = new DataStructureGenerator(this.minim);
+    const dataStructures = new Category();
+    dataStructures.classes.push('dataStructures');
+
+    _.forEach(definitions, (schema, key) => {
+      this.withPath(key, () => {
+        try {
+          const jsonSchema = convertSchema(schema, { definitions }, false);
+          const dataStructure = generator.generateDataStructure(jsonSchema);
+
+          if (dataStructure) {
+            dataStructure.content.id = idForDataStructure(`#/definitions/${key}`);
+
+            if (this.generateSourceMap) {
+              this.createSourceMap(dataStructure, this.path);
+            }
+
+            dataStructures.push(dataStructure);
+          }
+        } catch (error) {
+          // TODO: Expose errors once feature is more-complete
```

Comment:
This is the same TODO in other data structure generation. I'd like to remove it and produce a warning that it couldn't be generated. I will do that in separate PR for both cases.